README.md

```diff
@@ -66,16 +66,18 @@ To configure `restic-ftp-docker`, you can use
-| RESTIC_FTP_DOCKER_KEEP_MONTHLY         | `12`        | argument for `--keep-monthly` of `restic forget` command, see [restic forget policies](https://restic.readthedocs.io/en/latest/060_forget.html#removing-snapshots-according-to-a-policy) |
+| Name                               | Default     | Description                                                                                                                                                                              |
+| ---------------------------------- | ----------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| RESTIC_DOCKER_BACKUP_CRON_SCHEDULE | `0 * * * *` | Cron expression to launch [backup script](/backup.sh)                                                                                                                                    |
+| RESTIC_DOCKER_BACKUP_FOLDER        | `/backup`   | Backup path                                                                                                                                                                              |
+| RESTIC_DOCKER_IS_FORGET_ENABLE     | `1`         | Enable or disable `restic forget script in crontab`                                                                                                                                      |
+| RESTIC_DOCKER_FORGET_CRON_SCHEDULE | `0 0 * * *` | Cron expression to launch [forget script](/forget.sh)                                                                                                                                    |
+| RESTIC_DOCKER_KEEP_HOURLY          | `24`        | argument for `--keep-hourly` of `restic forget` command, see [restic forget policies](https://restic.readthedocs.io/en/latest/060_forget.html#removing-snapshots-according-to-a-policy)  |
+| RESTIC_DOCKER_KEEP_DAILY           | `7`         | argument for `--keep-daily` of `restic forget` command, see [restic forget policies](https://restic.readthedocs.io/en/latest/060_forget.html#removing-snapshots-according-to-a-policy)   |
+| RESTIC_DOCKER_KEEP_WEEKLY          | `4`         | argument for `--keep-weekly` of `restic forget` command, see [restic forget policies](https://restic.readthedocs.io/en/latest/060_forget.html#removing-snapshots-according-to-a-policy)  |
+| RESTIC_DOCKER_KEEP_MONTHLY         | `12`        | argument for `--keep-monthly` of `restic forget` command, see [restic forget policies](https://restic.readthedocs.io/en/latest/060_forget.html#removing-snapshots-according-to-a-policy) |
+
+Note: the `RESTIC_DOCKER_` prefix is used for `restic-ftp-docker` projet environment variables. When `_DOCKER_` is not present in the prefix then these are [environment variables supported in the restic project](https://restic.readthedocs.io/en/stable/040_backup.html#environment-variables).
```

Comment:
@Its-Alex what do you think about this note? Is it easy to understand?

Reply:
@stephane-klein I think you can merge that text with https://github.com/Its-Alex/restic-ftp-docker/pull/2/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R65

Reply:
@Its-Alex :+1: good idea.

Reply:
@Its-Alex Done.